apps-rendering/src/components/editions/galleryImage/index.tsx

```diff
@@ -49,7 +49,18 @@ const styles = css`
 const getCaptionDetails = (oDoc: Option<DocumentFragment>): CaptionDetails => {
 	const details: CaptionDetails = {
 		location: none,
-		description: none,
+		description: some([]),
```

Comment:
Just asking so that I can learn more typescript :) Should we not indicate what is the array type? Like is it string[] or number[] or etc?

Reply:
The `CaptionDetails` type on line 50 contains the `description` type: `Option<string>` :)

Reply:
Oh I see, so here we are instantiating the object by giving the description an empty array. Thanks Joe